core/providers/anthropic.go

```diff
@@ -798,17 +798,24 @@ func parseAnthropicResponse(response *AnthropicChatResponse, bifrostResponse *sc
 		}
 	}
 
+	// Build MessageContent based on the type of content blocks
+	messageContent := schemas.MessageContent{}
```

Comment:
@Pratham-Mishra04 we might need to use from pooling vs allocating from heap here right ?